var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -147,4 +147,11 @@ def nvariant_bool(feature):
                 '-DPARAVIEW_DO_UNIX_STYLE_INSTALLS:BOOL=ON',
             ])
 
+        # Hide git from Paraview so it will not use `git describe`
+        # to find its own version number
+        if self.spec.satisfies('@5.4.0:5.4.1'):
```

Comment:
there is `spec = self.spec` above, so you can write `spec.satisfies...` to make it slightly shorter